data_structures/static_rtree.hpp

```diff
@@ -642,11 +642,56 @@ class StaticRTree
         return result_coordinate.is_valid();
     }
 
+    /**
+     * Checks whether A is between B-range and B+range, all modulo 360
+     * e.g. A = 5, B = 5, range = 10 == true
+     *      A = -6, B = 5, range = 10 == false
+     *      A = -2, B = 355, range = 10 == true
+     *      A = 6, B = 355, range = 10 == false
+     *      A = 355, B = -2, range = 10 == true
+     *
+     * @param A the bearing to check, in degrees, 0-359, 0=north
+     * @param B the bearing to check against, in degrees, 0-359, 0=north
```

Comment:
Yep.  I'm being restrictive in the documentation, but forgiving in the implementation.  Because of edge cases around 0 degrees, it turned out to be easier to implement negative values anyway.  Docs are just to encourage users to keep it to 0-359.